### ISSUE TITLE
Auto renew?

### ISSUE THREAD

**Issue by ReenigneArcher** [hassio-addons/addon-nginx-proxy-manager#156]
# Problem/Motivation

> Is this supposed to auto renew my SSL certificates? My certificates are 6 days away from expiring. I would have expected them to renew at least 30 days before expiration.

## Expected behavior

> Certificates auto renew and I don't get emails from let's encrypt saying they are about to expire.

## Actual behavior

> They don't auto renew... I can renew manually.

## Steps to reproduce

> I don't know.

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by frenck** [comment]
Yes, the Nginx proxy manager auto-renews, manual intervention should not be needed.

**Comment by ReenigneArcher** [comment]
> 
> 
> Yes, the Nginx proxy manager auto-renews, manual intervention should not be needed.

Okay, so I had several certificates which the server was not running (the dns is pointing at nginx though) so I assumed these would still renew, but they do not.

Anyway the ones that were running apparently did renew, but the UI never updated until I tried to manually update. I did the manual update yesterday on these and it now says they will expire on April 17th, instead of February 16th. The others that were failing I fixed the proxies for today and renewed, those are saying they now expire on May 12th.
